test/test_genome.py

```diff
@@ -766,3 +767,77 @@ def test_ncolumns_zero(rng):
 
     CartesianGraph(genome).to_func()
     CartesianGraph(genome).to_numpy()
+
+
+def test_splice_dna(genome_params, rng):
+
+    genome = cgp.Genome(**genome_params)
+
+    with pytest.raises(ValueError):
+        [] = genome.splice_dna(dna_insert=[1, 2, 3])
```

Comment:
```suggestion
        genome.splice_dna(dna_insert=[1, 2, 3])
```
not sure what this does, but if you don't want to use the return value of a function you just don't assign it a name